CreeDictionary/search_quality/sample.csv

```diff
@@ -1,70 +1,384 @@
+dawn,pêtâpan,wâpan,,,
+wheel,wâsakâpayîs,,,,
+small,apisâsin,apisîsisiw,,,
+green,askihtakosiw,askihtakwan,,,
+short,cimisisiw,cimâsin,,,
+high,ispâw,,,,
+fast,kakwêyâhow,,,,
+clean,kanâtan,kanâtisiw,,,
+black,kaskitêsiw,kaskitêwâw,,,
+beautiful,katawasisin,katawasisiw,,,
+tall,kinosiw,kinwâw,,,
+strong,maskawisîw,maskihkîwimâkwan,,,
+red,mihkosiw,mihkwâw,,,
+big,misikitiw,misâw,,,
+nice,miyosiw,,,,
+good,miywâsin,,,,
+ugly,mâyâtan,mâyâtisiw,,,
+orange,osâwisiw,osâwâw,osâwâs,,
+grey,pihkonâkosiw,pihkonâkwan,,,
+broken,pîkopayin,pîkopayiw,,,
+blue,sîpihkosiw,sîpihkwâw,,,
+wine red,watôkimihkosiw,watôkimihkwâw,,,
+likes the taste,wîhkistêw,wîhkistam,,,
+dirty,wiyipâtan,wiyipâtisiw,,,
+pink,wâpikwanîwinâkosiw,wâpikwanîwinâkwan,,,
+white,wâpiskisiw,wâpiskâw,,,
+yellow,wâposâwisiw,wâposâwâw,,,
+delicious,wîhkasin,wîhkitisiw,,,
+purple,wîpi-sîpihkosiw,wîpi-sîpihkwâw,,,
+brown,wîposâwâw,wîpôsâwisiw,,,
+slow,yikicikâwîmakan,yîkicikâwîw,,,
+go away,awas,awasitik,,,
+just kidding,cwâ,,,,
+wait,cêskwa,,,,
+near,cîki,,,,
+that's okay,kiyâm,,,,
+ok,kiyâm,,,,
+always,kâkikê,,,,
+don't,kâya,,,,
+let's,mahti,,,,
+right now,mêkwâc,sêmâk,,,
+here,na,,,,
+because,osâm,,,,
+already,âsay,,,,
+come here,âstam,âstamitik,,,
+okay now,âw,,,,
+never mind,ekosi kîyâm,,,,
+look over there,itâpi nêtê,,,,
+well,êkosi mâka,,,,
+that's it for now,êkosi pita,,,,
+coffee,pihkahcêwâpôs,pihkahtêwâpoy,,,
+meat,wiyâs,,,,
+sit down,apiw,,,,
+has,ayâw,ayâwêw,,,
+looks,itâpiw,kitâpahtam,kitâpamêw,,
+cooks,kîsisam,kîsiswêw,piminawasow,,
+likes,miywêyihtam,,,,
+eats,mowêw,mîciw,,,
+bakes,nawacîw,,,,
+hits,pakamaham,pakamahwêw,,,
+gets up,pasikôw,,,,
+smiles,pâhpisiw,,,,
+hears,pêhtawêw,,,,
+gives a ride,pôsihêw,,,,
+rides,pôsiw,,,,
+blows on,pôtâtam,pôtâtêw,,,
+leaves,sipwêhtêw,nakasiwêw,nakatam,,
+touches,sâminam,sâminêw,,,
+sits on,têhtapiw,,,,
+gets up from bed,waniskâw,,,,
+moves,waskawîw,,,,
+goes outside,wayawîw,,,,
+since,aspin,,,,
+almost,kêkâc,,,,
+and a half,mîna âpihtâw,,,,
+from,ohci,,,,
+before,pâmwayês,,,,
+half,âpihtaw,,,,
+number,akihtâson,,,,
+minute,cipahikanis,,,,
+stove,kotawânâpisk,,,,
+clock,pîsimohkân,,,,
+hour,tipahikan,,,,
+counts,akihcikêw,akihtam,akimêw,,
+morning,kîkisêpâyâw,,,,
+day,kîsikâw,,,,
+evening,otâkosin,,,,
+waits,pêhow,,,,
+night,tipiskâw,,,,
+noon,âpihtâ-kîsikâw,,,,
+midnight,âpihtâ-tipiskâw,,,,
+amount,itahto,,,,
+only,katisk,piko,,,
+dollar,nêwopêhikan,,,,
+penny,pîwâpiskos,,,,
+safety pin,sakâskwahon,,,,
+it is Sunday,ayamihâwi-kîsikâw,,,,
+weather is,isiwêpan,,,,
+years old,itahtopiponêw,,,,
+costs,itakihtêw,itakisow,,,
+Saturday,nikotwâsiko-kîsikâw,,,,
+Wednesday,nisto-kîsikâw,,,,
+Friday,niyânano-kîsikâw,,,,
+Thursday,nêwo-kîsikâw,,,,
+Tuesday,nîso-kîsikâw,,,,
+wins,otahowêw,,,,
+Monday,pôn-âyamihêwi-kîsikâw,,,,
+earns,sôniyâhkêw,,,,
+earns money,sôniyâhkêw,,,,
+clear sky,wâsêskwan,,,,
+little star,acâhkos,,,,
+beaver,amisk,,,,
+paddle,apoy,,,,
+stone,asiniy,,,,
+star,acâhkos,atâhk,,,
+frog,ayîkis,,,,
+snow,kôna,,,,
+tree,mîtos,,,,
```

Comment:
tree: mîtos is one type of tree and should be returned in the search, but mistik is the more general term